membersrvc/ca/ca.go

```diff
@@ -34,7 +34,6 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3" // TODO: justify this blank import or remove
```

Comment:
@adecaro right. I misunderstood go-sqlite3 documentation, this black import is required and a direct package access is required for load data local infile support (https://github.com/go-sql-driver/mysql)